internal/api/grpc/user/v2/user.go

```diff
@@ -485,9 +408,98 @@ func (s *Server) HumanMFAInitSkipped(ctx context.Context, req *user.HumanMFAInit
 }
 
 func (s *Server) CreateUser(ctx context.Context, req *user.CreateUserRequest) (*user.CreateUserResponse, error) {
-	return nil, zerrors.ThrowUnimplemented(nil, "", "not implemented")
+	orgId := req.GetOrganizationId()
+	if err := s.command.CheckOrgExists(ctx, orgId); err != nil {
+		return nil, err
+	}
```

Comment:
Shouldn't this be part of the command?

Reply:
I'll have a look

Reply:
It looks like for the existing APIs, the org is already checked in an API middleware.
I' don't want to duplicate the checks, nor do I want to change a middleware in this PR.
Therefore I added optional checks to the add human and machine commands.
Unit tests are added, too.
https://github.com/zitadel/zitadel/pull/9763/commits/91c14038046b71cbee6aa0b4eba3ea8459d0ec82